packages/big-design/src/components/Datepicker/Datepicker.tsx

```diff
@@ -67,6 +67,15 @@ const RawDatepicker: React.FC<DatepickerProps & PrivateProps> = ({
         minDate={min ? new Date(min) : undefined}
         onChange={updateDate}
         placeholderText={placeholder}
+        popperModifiers={[
+          {
+            name: 'removeOffset',
+            fn: ({ y }) => ({
+              y: y - 10,
+            }),
+          },
+        ]}
+        popperPlacement="bottom-start"
```

Comment:
Noticed there was some styling issues with the v5 of `react-datepicker`. Fixed it in a commit here, POL:
![Screenshot 2024-04-16 at 11 49 06](https://github.com/bigcommerce/big-design/assets/10539418/d8087020-3694-47bf-940c-7b98e1f5675b)